Assets/MixedRealityToolkit/Services/BaseDataProviderAccessCoreSystem.cs

```diff
@@ -37,6 +47,7 @@ public override void Enable()
             foreach (var provider in dataProviders)
             {
                 provider.Enable();
+                if (isDestroyed) { break; }
```

Comment:
Why do we enable the first dataprovider and then check if I the service was destroyed we should stop?

Reply:
Same with others below